PartThree.js

```diff
@@ -2,21 +2,90 @@
 class PartThree {
 
     findYoungest(data){
-        return data;
+        let person;
+
+        let rows = getRows(data);
+
+        let smallest = 9999;
```

Comment:
Would recommend starting smallest at the age in the first record. If The lowest age is 10000, this wouldn't work